docs/_install-and-build/cross-compile.md

```diff
@@ -1,21 +1,22 @@
 ---
-title: Cross-compiling Ethereum
+title: Cross-Compiling Geth
+sort_key: C
 ---
-**Note: All of these and much more have been merged into the project Makefile.
-You can cross build via `make geth-<os>-<platform>` without needing to know any
-of these details from below.**
-
-Developers usually have a preferred platform that they feel most comfortable
-working in, with all the necessary tools, libraries and environments set up for
-an optimal workflow. However, there's often need to build for either a different
-CPU architecture, or an entirely different operating system; but maintaining a
-development environment for each and switching between the them quickly becomes
-unwieldy.
-
-Here we present a very simple way to cross compile Ethereum to various operating
-systems and architectures using a minimal set of prerequisites and a completely
-containerized approach, guaranteeing that your development environment remains
-clean even after the complex requirements and mechanisms of a cross compilation.
+
+**Note: All of these and much more have been merged into the project Makefile. You can
+cross build via `make geth-<os>-<platform>` without needing to know any of these details
+from below.**
```

Comment:
Don't we want to add a disclaimer here? I understand cross-compilation is a tricky beast.

Reply:
Cross compilation is actually very simple thanks to Peter's xgo tool.